src/modules/physics/box2d/wrap_Physics.cpp

```diff
@@ -477,12 +477,100 @@ int w_setMeter(lua_State *L)
 	return 0;
 
 }
+
 int w_getMeter(lua_State *L)
 {
 	lua_pushinteger(L, Physics::getMeter());
 	return 1;
 }
 
+int w_linearStiffness(lua_State* L) {
+	float frequency = (float)luaL_checknumber(L, 1);
+	float dampingRatio = (float)luaL_checknumber(L, 2);
+	Body* body1 = luax_checkbody(L, 3);
+	b2Body* other = 0;
+
+	if (lua_isnoneornil(L, 4)) {
```

Comment:
love's code style has braces on separate lines - and for situations like this where there's only one line inside an `if`, the braces can be completely removed.